src/components/FilterCollapse.js

```diff
@@ -36,7 +36,7 @@ const NumericField = ({ name, humanName, placeholder, valueOp, value, onChange,
       <option value=">">greater than</option>
       <option value="<=">less than or equal to</option>
       <option value=">=">greater than or equal to</option>
-      <option value="!">not equal to</option>
+      <option value="!=">not equal to</option>
```

Comment:
for not equal to, shouldn't we use '=' with '-' prepended to the entire term?

Reply:
yes, != doesn't work with the current filter syntax afaik. Or if it does, I have no idea how